plugins/browser-plugin-error-tracking/test/events.test.ts

```diff
@@ -126,4 +142,16 @@ describe('AdTrackingPlugin', () => {
       },
     });
   });
+
+  it('trackError replaces undefined messages with placeholder', () => {
+    expect(
+      extractUeEvent('iglu:com.snowplowanalytics.snowplow/application_error/jsonschema/1-0-1').from(state.outQueues[3])
+    ).toMatchObject({
+      schema: 'iglu:com.snowplowanalytics.snowplow/application_error/jsonschema/1-0-1',
+      data: {
+        message: 'y'.repeat(2048),
```

Comment:
I'm kind of surprised this doesn't fail with the above implementation?

Reply:
i think that was just by chance as the test string was greater than 8192, and was then truncated to the correct length!